packages/icons/src/social/IconInstagram.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import withIcon, { Props } from '../withIcon';
+
+function IconInstagram(props: Props) {
+  return (
+    <svg viewBox="0 0 64 64" xmlnsXlink="http://www.w3.org/1999/xlink" {...props}>
```

Comment:
Default is 64 compared to 16 of other icons. Does this cause issues?

Reply:
nope! since we apply `1em` by default. and the box is still square